notebooks/QA_evaluation_metrics.ipynb

```diff
@@ -0,0 +1,1271 @@
+{
```

Comment:
<p>Could we split the notebook here? In that case, we'd have one notebook that demos all the metrics and then the other one that works on inferencing and evaluation using those metrics?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/redhat-et/foundation-models-for-documentation/pull/16/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Reply:
<p>Sure!! </p>